### DISCUSSION TITLE
creating verifier taking infinite time

### DISCUSSION THREAD

**Discussion by infocodiste** [Web3Auth/Web3Auth#712] [Help]
I have chosen the environment as a Testnet and login provider to custom. it is showing processing for 4 hours.. ideally, it should be created within 20 minutes.

Thanks in advance..

**Comment by rinzler99** [comment]
Hey Thanks for reaching out. Team will look into the issue and get it resolved asap, meanwhile you share your verifier id / verifier name.

**Comment by tanguyenvn** [comment]
Hi, there is an issue with Ropsten Infura endpoint being deprecated. We have just resolved the issue. The verifier deployment should be fast now. Here is the incident summary https://status.web3auth.io/incidents/y5nfm10mcjmc
